db/migrations/20230103172725_dogs.js

```diff
@@ -1,7 +1,13 @@
 exports.up = knex =>
   knex.schema.createTable("dogs", tbl => {
     tbl.increments();
-    tbl.text("task", 128).notNullable();
+    tbl.text("breed", 128).notNullable();
+    tbl.text("size", 128).notNullable();
+    tbl.integer("family-rating", 128).notNullable();
+    tbl.integer("trainability", 128).notNullable();
+    tbl.integer("grooming-needs", 128).notNullable();
+    tbl.integer("energy-level", 128).notNullable();
```

Comment:
Hey @Mrcolin99! When you get a chance can you change `hyphens` to `underscores`? It's Postgres convention to use underscores in names. This will prevent us from having issues when we add our data next.